services/polymart/polymart-latest-version.service.js

```diff
@@ -0,0 +1,45 @@
+import { NotFound } from '../../core/base-service/errors.js'
+import { BasePolymartService, documentation } from './polymart-base.js'
+
+export default class PolymartLatestVersion extends BasePolymartService {
+  static category = 'version'
+
+  static route = {
+    base: 'polymart/version',
+    pattern: ':resourceId',
+  }
+
+  static examples = [
+    {
+      title: 'Polymart Version',
+      namedParams: {
+        resourceId: '323',
+      },
+      staticPreview: this.render({
+        version: '1.2.9',
+      }),
+      documentation,
+    },
+  ]
+
+  static defaultBadgeData = {
+    label: 'version',
+    color: 'blue',
+  }
+
+  static render({ version }) {
+    return {
+      message: `${version}`,
+    }
+  }
+
+  async handle({ resourceId }) {
+    const { response } = await this.fetch({ resourceId })
+    if (!response.resource) {
+      throw new NotFound()
+    }
+    return this.constructor.render({
```

Comment:
Same comment as https://github.com/badges/shields/pull/8352#discussion_r959829384
Can we use `renderVersionBadge()` here?

Reply:
Once we're doing that, we could also ditch the default colour in `defaultBadgeData` for this class.